.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+    cooldown:
+      default-days: 7
+    groups:
+      github-actions:
+        patterns:
+          - "*"
+
+  - package-ecosystem: cargo
```

Comment:
The dependabot cargo integration updates the lower bounds of dependencies in `Cargo.toml`. This works for applications, but for libraries we want wide version range and only bump the lockfile version (and a `-Zminimal-versions` run in CI, but that's a different task). We have the same problem in pubgrub where this stops us from merging dependabot PRs (https://github.com/pubgrub-rs/pubgrub/pull/346/files)

Reply:
Oh, great point. I'll remove this one then.